### PR TITLE
Use of Routers and React Basic Styling.

### PR DESCRIPTION
**Pull request by Graycemuthui**
### Description
In this project I achieved the following:
- Created a website consisting of 3 pages: Home, Calculator, and Quote. ✔️ 
-  Used React basic styling.  ✔️ 
- Used React-Router to make the routes work. ✔️ 
- Made sure that navigation links for all 3 routes are displayed on each page ✔️ 

### Changes Made
- Update the Pull request description
- Update the MIT file
- Fix the heading

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
-        <section className="app-container" />
-        <Calculator />
-      </div>
-    );
-  }
-}
+const App = () => (
+  <div className="App">
+    <Router>
+      <Nav />
```
Comment (medaminedev66) [inline src/App.js]: - [ ] Following the project requirements, The general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md):
- Please add the title of the website (Math magicians) to the header.
- On the calculator page, the text (Let's do Math) should be on the left and the calculator design should be on the right.

image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Graycemuthui  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- You have a descriptive PR ✔️

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- The `MIT link`  in the README file is broken. I would recommend fixing that to make README professional.
- Pull Request should have a descriptive title telling others what changes are inside. It would be better to update that so your project is documented professionally.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
```
Comment (danmainah) [inline src/App.js]: - [ ] @Graycemuthui great work implementing the changes requested, however, you have not still added the title, which should be on the top left corner as in the initial design as shown below:

 Design Given         |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Graycemuthui , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
# Status: Approved 🎉🎉🎉

Hi @Graycemuthui , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**